### DISCUSSION TITLE
[Question] is this available for playwright?

### DISCUSSION THREAD

**Discussion by searchingforcode** [QIN2DIM/hcaptcha-challenger#162] [General]
Hi is there any playwright compatible version?

**Comment by Vinyzu** [comment]
You can plug the AI Part into Playwright
Im doing a Discord Token Gen rn (using this and mouse emulation), it maight release the next days so...

**Comment by Vinyzu** [comment]
You can plug my playwright code easily

**Comment by Vinyzu** [comment]
> Hi is there any playwright compatible version? 

https://github.com/Vinyzu/DiscordGenerator

(Qin2Dim that shouldnt be an ad i really appreciaete ur work just wanna help out this folk because i think my code would be fitting for his needs ;:D)

**Comment by Vinyzu** [comment]
https://github.com/QIN2DIM/hcaptcha-challenger/discussions/214
